docs/setup/independent/install-kubeadm.md

```diff
@@ -135,7 +135,9 @@ You will install these packages on all of your machines:
 kubeadm **will not** install or manage `kubelet` or `kubectl` for you, so you will 
 need to ensure they match the version of the Kubernetes control panel you want 
 kubeadm to install for you. If you do not, there is a risk of a version skew occurring that
-can lead to unexpected, buggy behaviour.
+can lead to unexpected, buggy behaviour. However, _one_ minor version skew between the 
+kubelet and the control plane is supported. For example, kubelets running 1.7.0 should be 
+fully compatible with a 1.8.0 API server.
```

Comment:
...but the kubelet version may never exceed the API server version.

Reply:
I agree about the location split. How about we put all the version skew in the main section (where it currently lives) and link off to it after `unexpected, buggy behaviour`.